sdk/lib/arbiter/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+
+include_directories(${REACTOS_SOURCE_DIR}/sdk/lib/rtl)
+
+list(APPEND SOURCE
+     ${REACTOS_SOURCE_DIR}/ntoskrnl/io/arbiter/arbiter.c
```

Comment:
Why not have arbiter.c live in this directory and link ntoskrnl with arblib?

Reply:
I see that in the file "ntos.cmake" all files are within the boundaries of "${REACTOS_SOURCE_DIR}/ntoskrnl/"...

Reply:
Well then why the "arbiter.c" file would not be part of the compiled "arbiter" library that is in the SDK/lib ? (look e.g. at cmlib that is used also by ntoskrnl)